File: scripts/python/hello_world.py
#!/usr/bin/env python3
"""
Simple Hello World script for testing purposes.
"""


def greet(name="World"):
    """Return a greeting message."""
    return f"Hello, {name}!"


def main():
    """Main function."""
    print(greet())
    print(greet("Copilot"))


if __name__ == "__main__":
    main()

File: scripts/javascript/hello_world.js
#!/usr/bin/env node
/**
 * Simple Hello World script for testing purposes.
 */

function greet(name = "World") {
  return `Hello, ${name}!`;
}

function main() {
  console.log(greet());
  console.log(greet("Copilot"));
}

if (require.main === module) {
  main();
}

module.exports = { greet };
